### ISSUE TITLE
Possible missing agent.db

### ISSUE THREAD

**Issue by angrylinuxuser** [stevschmid/track-o-bot#150]
Hi,
since im using linux this might be a false report however i have tested this with wine and virtualbox.
I think for some time now there is no agent.db ( there is product.db but it looks like a binary file not json) file present so Hearthstone::ReadAgentAttribute wont return any good results. I think that Agent.db was replaced with Battle.net.config in  \user profile \appdata \ battle.net. 

Am i wrong?

**Comment by stevschmid** [comment]
You are absolutely right! The code is only there for fallback purposes (older clients still have a agent.db). I will remove it in the near future so I will keep this issue open to remind me.
